### DISCUSSION TITLE
why do streams  NOT LOAD/play in certain softwares like smplayer or bsplayer but vlc

### DISCUSSION THREAD

**Discussion by Cootmaster** [smplayer-dev/smplayer#1053] [General]
hi i have a few  music streams  and i noticed VLC, and WMP  play them , even my own radio software.
When i play here  or BS player  IT WONT PLAY  and will freeze up a bit
the url is  https://pub0302.101.ru:8000/stream/pro/aac/64/142  
do these streams not load and play in some players?

**Comment by saberbholt** [comment]
mmmm i have no isues in windows 11 23h2 with that url in my SMPlayer, i use that version:

SMPlayer windows 24.5.0 (revisión 10283) (64-bit) devoloper version
Qt 5.6.3 (compilado con Qt 5.6.3)
MPV v0.37.0-522-ga56d8ff1

In the new windows media  version win11.2406.13.0 no problem 
And VLC last version no isues too..

**Comment by Cootmaster** [comment]
![image1](https://github.com/user-attachments/assets/99299af8-23e7-430f-88e1-d2917ef825e6)
here is mine  
24.5.0 re 10277  win 10   see its stuck wont do anything
